.github/workflows/build-tasks.yml

```diff
@@ -22,13 +22,12 @@ jobs:
           - label: amd64
             runner: ubuntu-24.04
           - label: arm64
-            runner: buildjet-2vcpu-ubuntu-2204-arm
+            runner: ubuntu-22.04-arm
 
     runs-on: ${{ matrix.build.runner }}
     timeout-minutes: 10
 
     steps:
-      # NB: no podman on buildjet arm runners
```

Comment:
but this doesn't switch to podman and we surely would like too? :) Either way can be a follow up :)

Reply:
No I mean the moving the docker actions to podman as github runners support that.